google-cloud/docs/toc.json

```diff
@@ -55,6 +55,28 @@
         }
       ]
     },
+    {
+      "title": "Bigquery DataTransfer",
+      "type": "google/cloud/bigquery/data_transfer",
```

Comment:
@landrito These `type` values are consumed by the Angular site app and should not have underscores:  `datatransfer`, not `data_transfer`.

Please change throughout this JSON fragment.

Reply:
Done.